.github/workflows/template_gitops.yml

```diff
@@ -44,18 +44,29 @@ on:
         required: false
       gonosumdb:
         required: false
+      app-id:
+        required: false
+      private-key:
+        required: false
 
 jobs:
   gitops:
-
     name: GitOps
     runs-on: ubuntu-22.04
 
     if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/')
 
+    env:
+      USING_APP_CREDENTIALS: ${{ secrets.app-id != '' && secrets.private-key != '' }}
+
     steps:
-      - name: Checkout
```

Comment:
Not sure why we need the `actions/checkout` here

Reply:
If you want to start the action, you want to get the code of the repo where you execute this action. In this repo, you have the Dockerfile, etc. Without this action, you will get a not-found exception for several files. 😆 If you are unsure, feel free to test your solution. You can use the branch name as your new version. 🙌